core/src/display_object.rs

```diff
@@ -1180,6 +1429,7 @@ pub trait TDisplayObject<'gc>:
     /// Values other than the default `BlendMode::Normal` implicitly cause cache-as-bitmap behavior.
     fn set_blend_mode(&self, gc_context: MutationContext<'gc, '_>, value: BlendMode) {
         self.base_mut(gc_context).set_blend_mode(value);
+        self.invalidate_cached_bitmap(gc_context);
```

Comment:
An above comment references a bug in Flash where the bitmap is not invalidated when the blend mode is changed- would it be helpful to add that as a comment here (in case of some obscure edge case)?